Source/Core/VideoBackends/Vulkan/main.cpp

```diff
@@ -247,19 +247,7 @@ bool VideoBackend::Initialize(void* window_handle)
       !PerfQuery::GetInstance()->Initialize())
   {
     PanicAlert("Failed to initialize Vulkan classes.");
-    g_perf_query.reset();
-    g_texture_cache.reset();
-    g_vertex_manager.reset();
-    g_renderer.reset();
-    StateTracker::DestroyInstance();
-    g_framebuffer_manager.reset();
-    g_shader_cache->Shutdown();
-    g_shader_cache.reset();
-    g_object_cache.reset();
-    g_command_buffer_mgr.reset();
-    g_vulkan_context.reset();
-    ShutdownShared();
-    UnloadVulkanLibrary();
+    Shutdown();
```

Comment:
If we're going to do this, shouldn't we be calling Shutdown() in _all_ the 'false' cases?

Reply:
(or maybe if we return false in `Initialize` we call shutdown then?)

Reply:
+1 on this, should be safe to do.

Reply:
Good idea, I've replaced `ShutdownShared(); UnloadVulkanLibrary();` with calls to `Shutdown();`.